src/core/Akka.Streams.Tests/Implementation/Fusing/InterpreterSupervisionSpec.cs

```diff
@@ -261,80 +261,6 @@ public void Interpreter_error_handling_should_fail_when_OnPull_throws()
         }
 
         [Fact]
-        public void Interpreter_error_handling_should_resume_when_Filter_throws()
```

Comment:
These specs are moved into the corresponding DSL specs (FlowScanSpec, FlowWhereSpec...)